Samples/Mica/cpp-win32/WinAppSDKMicaSample/WinAppSDKMicaSample.vcxproj

```diff
@@ -195,7 +195,7 @@
 
   <!-- Each release of Visual Studio produces larger intermediate files.
        To prevent build agents from running out of disk space, clean as we go. -->
-  <Target Name="CleanIntermediateFiles" AfterTargets="Build">
-    <RemoveDir Directories="$(IntDir)" />
-  </Target>
+  <!-- <Target Name="CleanIntermediateFiles" AfterTargets="Build">
+    <RemoveDir Directories="$(IntDir)" /> 
+  </Target> -->
```

Comment:
This Target seems to have been manually commented out. Should this be removed, or should the commenting out be reverted so that the Target exists again?

Reply:
I just realized this change has happened in the Mica sample. @samanthamsong - is this intentional? I'd expect all changes in your branch to be only for the Windowing samples.